doc/command_ref.rst

```diff
@@ -141,7 +141,15 @@ Options
     Show the help.
 
 ``--installroot=<path>``
-    set install root
+    Specifies an alternative installroot, relative to where all packages will be
+    installed. Think of this like doing ``chroot <root> dnf`` except using
+    ``--installroot`` allows dnf to work before the chroot is created. The
+    following things are taken from or stored in installroot: cachedir, gpgkey,
+    config file, log files, releasever, repository, pluginconffile. Note: You
```

Comment:
...config file (unless you use -c option, then the path is NOT relative to installroot) ...